voice/about.md

```diff
@@ -12,33 +12,40 @@
+| [`<SpeakSentence>`](bxml/verbs/speakSentence.md)     | The SpeakSentence verb converts text into audible speech.                                             |
+| [`<StartRecording>`](bxml/verbs/startRecording.md)   | The StartRecording verb allows a segment of a call to be recorded while other verbs are executing.    |
+| [`<StopRecording>`](bxml/verbs/stopRecording.md)     | The StopRecording verb stops a recording that was previously started by a `<StartRecording>`.         |
+| [`<Transfer>`](bxml/verbs/transfer.md)               | The Transfer verb is used to transfer the call to another number.                                     |
 
 ## BXML Callbacks
 
-| Callback                                                | Description                                                                    |
-|:--------------------------------------------------------|:-------------------------------------------------------------------------------|
-| [Answer](bxml/callbacks/answer.md)                      | Bandwidth API sends this to the application when the call is answered.         |
-| [Gather](bxml/callbacks/gather.md)                      | Bandwidth API sends this when a `<Gather>` is completed.                       |
-| [Initiate](bxml/callbacks/initiate.md)                  | Bandwidth API sends this to the application when an inbound call arrives.      |
+| Callback                                                | Description                                                                                                                                                                                            |
+|:--------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Answer](bxml/callbacks/answer.md)                      | Bandwidth API sends this to the application when the call is answered.                                                                                                                                 |
+| [Gather](bxml/callbacks/gather.md)                      | Bandwidth API sends this when a `<Gather>` is completed.                                                                                                                                               |
+| [Initiate](bxml/callbacks/initiate.md)                  | Bandwidth API sends this to the application when an inbound call arrives.                                                                                                                              |
+| [Record Complete](bxml/callbacks/recordComplete.md)     | Bandwidth API sends this to the application when a `<Record>` is completed. To download the recording look for the [Recording Available](bxml/callbacks/recordingAvailable.md) event.                  |
```

Comment:
```suggestion
| [Record Complete](bxml/callbacks/recordComplete.md)     | Bandwidth API sends this to the application when a `<Record>` is completed. To download a completed recording, utilize its [Recording Available](bxml/callbacks/recordingAvailable.md) event.                  |
```

Reply:
I think it would be better to say "The recording is not available to download until the Recording Available event is received.